engine/mission.py

```diff
@@ -63,14 +61,6 @@ def __init__(self, robot, base_station, init_control_mode, grid=Grid(42.444250,
         self.inactive_waypoints = self.grid.get_inactive_waypoints_list()
         self.waypoints_to_visit = deque(self.all_waypoints)
         self.allowed_dist_error = allowed_dist_error
-        if not robot.is_sim:
-            self.gps_serial = serial.Serial('/dev/ttyACM0', 19200, timeout=5) 
-            self.robot_radio_serial = serial.Serial('/dev/ttyS0', 57600) #robot radio 
-            self.imu_i2c = busio.I2C(board.SCL, board.SDA)
-            self.motor_controller = MotorController(robot, wheel_r = 0, vm_load1 = 1, vm_load2 = 1, L = 0, R = 0)
```

Comment:
wheel_radius instead of wheel_r?

the compilation is getting the following error
TypeError: MotorController.__init__() got an unexpected keyword argument 'wheel_r'